netlify-plugin-cloudinary/src/index.ts

```diff
@@ -277,6 +275,7 @@ export async function onBuild({
           }
 
           mediaPaths.forEach(async mediaPath => {
+            mediaPath = mediaPath.split(path.win32.sep).join(path.posix.sep);
```

Comment:
just so im totally understanding what's happening here, basically, in the event that someone is on a windows machine, it will find the separator, then reconstruct it with posix separators, so that the rest of the code works as expected, but if its not windows, it basically wont do anything since it won't be there

and windows separators are `\`?

right?